client/js/components/show-hide.js

```diff
@@ -0,0 +1,42 @@
+const showHide = (state) => {
+  const el = state.el;
+  const activeClass = state.activeClass || 'is-active';
+  const trigger = el.querySelector('.js-show-hide-trigger');
+  const drawer = el.querySelector('.js-show-hide-drawer');
+
+  const init = () => {
+    el.setAttribute('aria-haspopup', 'true');
```

Comment:
I was looking for the recommendation and can't find it - could you pass it on? Makes sense though.